src/assets/styles/layout.scss

```diff
@@ -81,6 +81,11 @@
   }
 }
 
+// Data table column helper class
+.column-lg {
+  width: 15rem;
+}
```

Comment:
Please move this to the (single) component where it's used.

If multiple components declare the same class style then I'm in favour of moving it to base.scss or layout.scss.

Reply:
Wait -- old code used `column-md`... where did that come from? Vuetify?

Reply:
It had no definition in this project yet, not a Vuetify class. 
These are defined in Assets for width consistencies in the many tables etc
The intention was to make it global for the same purpose but totally fine with putting it local to the Credentials table.
Updated!